src/index.html

```diff
@@ -8,6 +8,78 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+
+    <div class="stars stars--0">
+      <span class="inactive stars__star">
+      </span>
+      <span class="inactive stars__star">
+      </span>
+      <span class="inactive stars__star">
+      </span>
+      <span class="inactive stars__star">
+      </span>
+      <span  class="inactive stars__star">
+      </span>
+    </div>
+    <div class="stars stars--1">
+      <span class=" active stars__star">
+      </span>
+      <span class="inactive stars__star">
+      </span>
+      <span class="inactive stars__star">
+      </span>
+      <span class="inactive stars__star">
+      </span>
+      <span class="inactive stars__star">
+      </span>
```

Comment:
```suggestion
      <span class="stars__star">
      </span>
      <span class="stars__star">
      </span>
      <span class="stars__star">
      </span>
      <span class="stars__star">
      </span>
      <span class="stars__star">
      </span>
```
Remove this classes everywhere use `nth-child` instead